AstarteDeviceSDKCSharp/Protocol/AstarteDeviceAggregateDatastreamInterface.cs

```diff
@@ -42,7 +43,15 @@ public async Task StreamData(string path, Dictionary<string, object> payload, Da
                 throw new AstarteTransportException("No available transport");
             }
 
-            await transport.SendAggregate(this, path, payload, timestamp);
+            if (transport is AstarteTransportOffline)
+            {
+                await transport.SendAggregate(this, path, payload, timestamp);
+            }
+            else
+            {
+                await transport.SendAggregate(this, path, payload, timestamp);
+            }
```

Comment:
The AstarteTrasport is injected from outside, the interface should not whether the transport is `Offline` or not .
```suggestion
                await transport.SendAggregate(this, path, payload, timestamp);
```